src/pubsub/mod_pubsub_db_rdbms.erl

```diff
@@ -86,6 +83,146 @@
+        <<"DELETE FROM pubsub_items WHERE nidx = ? AND itemid IN (?)">>),
+
+    % ------------------- Nodes --------------------------------
+    mongoose_rdbms:prepare(pubsub_update_pubsub_node, pubsub_nodes, [type, owners, options, nidx],
+        <<"UPDATE pubsub_nodes SET type = ?, owners = ?, options = ? WHERE nidx = ?">>),
+    PubsubNodeFields = pubsub_node_fields(),
+    mongoose_rdbms:prepare(pubsub_select_node_by_key_and_name, pubsub_nodes, [key, name],
+        <<"SELECT ", PubsubNodeFields/binary, " from pubsub_nodes WHERE p_key = ? AND name = ?">>),
+    mongoose_rdbms:prepare(pubsub_select_node_by_id, pubsub_nodes, [nidx],
+        <<"SELECT ", PubsubNodeFields/binary, " from pubsub_nodes WHERE nidx = ?">>),
+    mongoose_rdbms:prepare(pubsub_select_nodes_by_key, pubsub_nodes, [key],
+        <<"SELECT ", PubsubNodeFields/binary, " from pubsub_nodes WHERE p_key = ?">>),
+    mongoose_rdbms:prepare(pubsub_select_nodes_in_list_with_key, pubsub_nodes, [key, names],
+        <<"SELECT ", PubsubNodeFields/binary, " from pubsub_nodes "
+          "WHERE p_key = ? AND name IN (?)">>),
+    PubsubNodeFieldsPrefixed = pubsub_node_fields("pn"),
+    mongoose_rdbms:prepare(pubsub_select_nodes_by_affiliated_user,
+        pubsub_affiliations, [luser, lserver],
+        <<"SELECT aff, ", PubsubNodeFieldsPrefixed/binary, " FROM pubsub_affiliations AS pa "
+          "INNER JOIN pubsub_nodes AS pn ON pa.nidx = pn.nidx WHERE luser = ? AND lserver = ?">>),
+    mongoose_rdbms:prepare(pubsub_select_subnodes_empty_node, pubsub_nodes, [key],
+        <<"SELECT ", PubsubNodeFieldsPrefixed/binary, " from pubsub_nodes as pn "
+          "LEFT JOIN pubsub_node_collections as collection ON pn.name = collection.name "
+          "WHERE p_key = ? AND collection.parent_name IS NULL">>),
+    mongoose_rdbms:prepare(pubsub_select_subnodes, pubsub_nodes, [parent_name, key],
+        <<"SELECT ", PubsubNodeFieldsPrefixed/binary, " from pubsub_nodes as pn "
+          "INNER JOIN pubsub_node_collections as collection ON pn.name = collection.name AND "
+          "collection.parent_name = ? WHERE p_key = ?">>),
+    mongoose_rdbms:prepare(pubsub_delete_node, pubsub_nodes, [key, name],
+        <<"DELETE from pubsub_nodes WHERE p_key = ? AND name = ?">>),
+    mongoose_rdbms:prepare(pubsub_del_parents, pubsub_node_collections, [node],
+        <<"DELETE FROM pubsub_node_collections WHERE name = ?">>),
+
+    % ------------------- GDPR --------------------------------
+    mongoose_rdbms:prepare(pubsub_get_user_items, pubsub_items, [created_luser, created_lserver],
+        <<"SELECT name, itemid, payload FROM pubsub_items INNER JOIN pubsub_nodes "
+          "ON pubsub_items.nidx = pubsub_nodes.nidx WHERE created_luser = ? "
+          "AND created_lserver = ?">>),
+    mongoose_rdbms:prepare(pubsub_select_nodes_by_owner_mysql, pubsub_nodes, [ljid],
+        <<"SELECT name, type FROM pubsub_nodes WHERE owners = convert([\"?\"], JSON);">>),
```

Comment:
that is something weird.

Probably should be
```
<<"SELECT name, type FROM pubsub_nodes WHERE owners = convert([?], JSON);">>
```

Also, that query is very slow and we would have to get rid of this mole (maybe it's not used or there is more efficient way to do that).